examples/Bunch/include/simulation_defines/param/memory.param

```diff
@@ -53,4 +53,8 @@ constexpr uint32_t BYTES_EXCHANGE_Y = 6 * 512 * 1024; //6 MiB
 constexpr uint32_t BYTES_EXCHANGE_Z = 4 * 256 * 1024; //4 MiB
```

Comment:
A little off-topic but just because I'm seeing it right now: in many `memory.param` files of the different examples there are different sizes of memory that need to be reserved for exchange but the comments often read the same numbers.
Just for consistency we could remove the comments there and state in the beginning that the unit used there is *Byte*.
Here at this point it would be 1 rather than 4 MiBs, right?